src/test/acceptance/features/unsuccessful-application.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Unsuccessful application
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  If my claim is unsuccessful I want to be shown the why my claim was unsuccessful
```

Comment:
`the reason why` or just `why`?

Reply:
Changed to 'If my claim is unsuccessful I want feedback about why my claim was unsuccessful'